### PR TITLE
Add an enum for the prompt commands

### PR DESCRIPTION
**Pull request by wesleywiser**
Part of #31

### CODE REVIEW

**Comment by wesleywiser** [comment]
Is this what you were thinking of in #31? I wanted to get your feedback and make sure I was on the right path before I started working on the rest of the task.

File: src/core/prompt.rs [hunk 1]
```diff
@@ -5,54 +5,134 @@ use edit::buffer::{TextBuffer, SplitBuffer};
 
 use std::process::exit;
 
-impl Editor {
-    /// Invoke a command in the prompt
-    pub fn invoke(&mut self, cmd: String) {
-        let mut split = cmd.split(' ');
+/// Prompt mode commands
```
Comment (ticki) [inline src/core/prompt.rs]: Could you add dots in the end of the doc comments?

**Comment by ticki** [comment]
This is perfect! Thanks. I left one nitty.

**Comment by wesleywiser** [comment]
Got it

**Comment by ticki** [comment]
:shipit:

**Pull request merged**